source/includes/_zapier.md

```diff
@@ -0,0 +1,37 @@
+## LOCALiQ App on Zapier®
+
+**Summary:** Zapier® is a tool that allows two systems to exchange data without any coding. LOCALiQ makes available a lead app on the Zapier® platform. You can reach out to [LOCALiQ API Support](mailto:apiservices@localiq.com) to request access.
+
+- The LOCALiQ Zapier® app can be used at no additional cost.
+- Zapier® offers a free trial if you would like to test this solution.
+- The setup can be be completed by client or CSM.
+
+NOTE: There is an additional monthly subscription cost through Zapier® The cost depends on the number of leads sent per month and the number of leads transmitted between the systems (which Zapier® refers to as "Tasks"). Pricing information for Zapier® can be found [here](https://zapier.com/pricing).
+
+**Parameters:**
+
+With the LOCALiQ App you can get contacts and contacts interactions for ingestion into the CRM of your choice. The CRM should also have an app available on Zapier®.
+
+Contacts: This HIPAA Compliant endpoint supports integration with CRM products. It returns all contacts (first name, last name, email and/or phone number) generated by a campaign for an advertiser. The response is organized by campaigns with the contacts nested within. Please note that the response also includes the associated contact and its meta data.
```

Comment:
I don't think this is quite what the webhook returns.  It delivers them one at a time as they are created in near-real-time.  I think you may be confusing the payload with what's returned in the Contacts API call.

Reply:
I had copied this from the Contacts API/Contact Interactions API sections per Jayanthi's instructions, but I agree that it didn't apply here. I simplified it based on your comment, but let me know if you want this section revised.